app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -179,6 +179,8 @@ export default {
     window.addEventListener('beforeunload', this.beforeWindowUnload);
 
     this.reRenderFormIo += 1;
+    const formStore = useFormStore();
+    formStore.setFormId(this.form.id);
```

Comment:
I am using the form store here to pass the formId from the Form Viewer to the Print Options. I couldn't find an alternative to do this so please let me know if you have some suggestions.

Reply:
The form id should already be set by the parent components, we fetch it and set it many times. It should work to just get the form from the form store and get the id from the form object inside of PrintOptions. Upon testing though, it seems like you would need to create a prop inside of `PrintOptions` maybe call it `f` for form id then create a computed value called `formId` that returns `this.f ? this.f : this.form.id`. Then in `FormViewerActions` you just need to pass the `formId` prop to the `PrintOptions` component so it would look like:

```
<PrintOptions
     :f="formId"
...
```
This would allow you to obtain the form id inside of `PrintOptions` without creating another variable for storing the form id. It'll get confusing in the future if we have multiple places where we store the form id.

Reply:
Thank you so much!